WordPress/Classes/Utility/Delay.swift

```diff
@@ -0,0 +1,96 @@
+import Foundation
+
+typealias Action = () -> Void
+
+/// Something that can be canceled
+///
+protocol Cancelable {
+    func cancel()
```

Comment:
I think this can go now. In a previous iteration it was useful, mostly for testing. But in the current version of the code I can't really justify it.